components/gc-follow-us/deprecated/follow-us-fr.html

```diff
@@ -26,13 +26,13 @@ <h2>Exemple de la composante Suivez-nous</h2>
 
 <section>
 	<h2>Exemple des icônes de médias sociaux individuels</h2>
-	<h3 class="icon twitter" lang="en">Twitter:</h3>
+	<h3 class="icon twitter" lang="en">X:</h3>
```

Comment:
The language tag is not needed here.

```suggestion
	<h3 class="icon x-social">X:</h3>
```